source/_posts/2024-12-19-voice-chapter-8-assist-in-the-home.markdown

```diff
@@ -0,0 +1,129 @@
+---
+layout: post
+title: "Voice Chapter 8 - Assist in the home today"
+description: "All the things you can do with Assist today, along with the state of our development, limitations, and where you can help."
+date: 2024-12-19 00:00:01
+date_formatted: "December 19, 2024"
+author: Mike Hansen
+comments: true
+categories: Assist
+og_image: /images/blog/2024-12-voice-chapter-8/art.png
+---
+
+<img src='/images/blog/2024-12-voice-chapter-8/art.png' alt="Voice chapter 8 - Assist in the home today">
+
+As you have probably already read, we launched our [Home Assistant Voice Preview Edition](/voice-pe/) today. The culmination of the past several years of open-source software progress on Home Assistant's home-grown voice assistant, [Assist](/voice_control/). A sizable group of dedicated developers has been working together on adding and honing its many features, and if it's been a while since you tried Assist you should use this launch as a chance to jump back in and see the progress we've made.
```

Comment:
```suggestion
As you have probably already read, we launched our [Home Assistant Voice Preview Edition](/voice-pe/) today. The culmination of the past several years of open-source software progress on Home Assistant's home-grown voice assistant, [Assist](/voice_control/). A sizable group of dedicated developers has been working together on adding and honing its many features, and if it's been a while since you tried Assist, you should use this launch as a chance to jump back in and see the progress we've made.
```